shells/pipes-shell/source/verbs/run-arc.js

```diff
@@ -49,73 +51,76 @@ export const runArc = async (msg, bus, runtime, defaultStorageKeyPrefix) => {
 };
 
 const instantiateRecipe = async (arc, runtime, recipe, particles) => {
-  let plan = await runtime.resolveRecipe(arc, recipe);
-  if (!plan) {
-    warn(`failed to resolve recipe ${recipe}`);
-    return false;
-  }
-  if (matchesRecipe(arc.activeRecipe, plan)) {
-    log(`recipe ${recipe} is already instantiated in ${arc}`);
-    for (const particle of particles) {
-      if (!reinstantiateParticle(arc, particle.id, particle.name)) {
-        return false;
-      }
-    }
-    return true;
-  }
+  // let plan = await runtime.resolveRecipe(arc, recipe);
+  // if (!plan) {
+  //   warn(`failed to resolve recipe ${recipe}`);
+  //   return false;
+  // }
 
-  for (const particle of particles) {
-    plan = updateParticleInPlan(plan, particle.id, particle.name, particle.providedSlotId);
-    if (!plan) {
-      warn(`failed updating particle id '${particle.id}', name ${particle.name} in recipe ${recipe}`);
-      return false;
-    }
-  }
+  // Note: Arc `reinstantiate` support was deprecated and removed.
+
+  // if (matchesRecipe(arc.activeRecipe, plan)) {
+  //   log(`recipe ${recipe} is already instantiated in ${arc}`);
+  //   for (const particle of particles) {
+  //     if (!reinstantiateParticle(arc, particle.id, particle.name)) {
+  //       return false;
+  //     }
+  //   }
+  //   return true;
+  // }
+
+  // for (const particle of particles) {
+  //   plan = updateParticleInPlan(plan, particle.id, particle.name, particle.providedSlotId);
+  //   if (!plan) {
+  //     warn(`failed updating particle id '${particle.id}', name ${particle.name} in recipe ${recipe}`);
+  //     return false;
+  //   }
+  // }
```

Comment:
we can't check this in all commented out - what's the deal here?

Reply:
these are remainders of the DevEx functionality that are no longer supported by the TS runtime. I think we no longer maintain pipes-shell anyway. I removed the commented out code.